src/server/engine_shard_set.cc

```diff
@@ -58,9 +58,10 @@ ABSL_FLAG(float, mem_defrag_page_utilization_threshold, 0.8,
           "this, memory in this page will defragmented");
 
 ABSL_FLAG(string, shard_round_robin_prefix, "",
-          "When non-empty, keys with hash-tags, whose hash-tag starts with this prefix are not "
+          "When non-empty, keys without hash-tags and keys with hash-tags, whose hash-tag starts "
```

Comment:
I think the description is a bit confusing.
Maybe write 
When non-empty keys which start with this prefix or keys with hash-tags whose hash-tag starts with this prefix